instrumentation/anr/src/main/java/io/opentelemetry/android/instrumentation/anr/AnrWatcher.kt

```diff
@@ -51,14 +67,36 @@ internal class AnrWatcher(
         }
         if (anrCounter.incrementAndGet() >= 5) {
             val stackTrace = mainThread.stackTrace
-            recordAnr(stackTrace)
+            emitAnrEvent(stackTrace)
             // only report once per 5s.
             anrCounter.set(0)
         }
     }
 
-    private fun recordAnr(stackTrace: Array<StackTraceElement>) {
-        val context = instrumenter.start(Context.current(), stackTrace)
-        instrumenter.end(context, stackTrace, null, null)
+    private fun emitAnrEvent(stackTrace: Array<StackTraceElement>) {
+        val attributesBuilder =
+            Attributes
+                .builder()
+                .put(THREAD_ID, mainThread.id)
+                .put(THREAD_NAME, mainThread.name)
+                .put(EXCEPTION_STACKTRACE, stackTraceToString(stackTrace))
+
+        for (extractor in additionalExtractors) {
+            extractor.onStart(attributesBuilder, Context.current(), stackTrace)
+        }
+
+        val eventBuilder = anrLogger.logRecordBuilder() as ExtendedLogRecordBuilder
+        eventBuilder
+            .setEventName("device.anr")
```

Comment:
We should explicitly call this out as a name change -- even if there isn't yet semconv around this. Specifically, the previous span name was called `ANR` and the new event name will be `device.anr`. It doesn't have to be in this PR, but I think we should follow this up with an effort to land a semantic convention for this. As part of that, I would also make the case that this probably belongs in the `app.` namespace, because it literally unpacks to "application not responding", which is more accurate than `device.`. Again, doesn't need to happen here. I just wanted to make sure this was noted.